### PR TITLE
Add Laravel Horizon link to CP

### PR DESCRIPTION
**Pull request by michaelr0**
If Horizon is installed and the current user can access it, then add a link to Horizon in the control panel (where or to the right of the telescope link would be/is)

Not sure which SVG to use.

### CODE REVIEW

File: resources/views/partials/global-header.blade.php [hunk 1]
```diff
@@ -35,6 +35,11 @@
                 @cp_svg('telescope')
             </a>
         @endif
+        @if (Route::has('horizon.index') && $user->can('viewHorizon'))
+            <a class="hidden md:block h-6 w-6 p-sm text-grey ml-2 hover:text-grey-80" href="/{{ route('horizon.index') }}" target="_blank" v-tooltip="'Laravel Horizon'">
+                @cp_svg('charts')
```
Comment (duncanmcclean) [inline resources/views/partials/global-header.blade.php:40]: Regarding which SVG to use, I wonder if it's possible to pull in the actual logo for Horizon, similar to what we're doing for Telescope.

Comment (michaelr0) [reply resources/views/partials/global-header.blade.php:40]: Thanks Duncan, I wasn't really sure how to go about bringing that in and if there would be any permission issues in doing so.

Comment (duncanmcclean) [reply resources/views/partials/global-header.blade.php:40]: I don't think there'd be a permissions issue - we already do it for Telescope so 🤷‍♂️ 

Should be able to use this one: https://github.com/laravel/laravel.com-next/blob/master/public/assets/svg/logo-horizon.svg

Comment (michaelr0) [reply resources/views/partials/global-header.blade.php]: Great, thanks @duncanmcclean 
I have added that now.

**Review by jasonvarga — APPROVED** [review]

**Pull request merged**